scripts/release_support/github-release.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
Please rename this file to `create-cortx-github-release` to make it for CORTX specific

Reply:
Done and tested